pkg/plugins/ansible/v1/scaffolds/internal/templates/config/rbac/role.go

```diff
@@ -39,7 +39,7 @@ func (f *ManagerRole) SetTemplateDefaults() error {
 	}
 
 	f.TemplateBody = fmt.Sprintf(roleTemplate, machinery.NewMarkerFor(f.Path, rulesMarker))
-
+	f.IfExistsAction = machinery.OverwriteFile
```

Comment:
Added this line due to: https://github.com/kubernetes-sigs/kubebuilder/pull/3627